src/freenet/clients/http/PproxyToadlet.java

```diff
@@ -602,8 +610,12 @@ private void showOfficialPluginLoader(ToadletContext toadletContext, HTMLNode co
           continue;
         }
 				HTMLNode pluginNode = pluginGroupNode.addChild("div", "class", "plugin");
-				HTMLNode option = pluginNode.addChild("input", new String[] { "type", "name", "value" }, new String[] { "radio", "plugin-name", pluginDescription.name });
-				option.addChild("i", pluginDescription.getLocalisedPluginName());
+				HTMLNode option = pluginNode.addChild("input",
+					new String[] { "type", "name", "value", "id" },
+					new String[] { "radio", "plugin-name", pluginDescription.name, "radioPlugin" + pluginDescription.name });
+				option.addChild("label",
+					new String[] { "for" },
+					new String[] { "radioPlugin" + pluginDescription.name }).addChild("i", pluginDescription.getLocalisedPluginName());
```

Comment:
could you add a newline before `.addChild("i")`?

Reply:
Would it be better like
```java
option.addChild("label",
  new String[] { "for" },
  new String[] { "radioPlugin" + pluginDescription.name })
  .addChild("i", pluginDescription.getLocalisedPluginName());
```
or rather like
```java
option.addChild("label",
  new String[] { "for" },
  new String[] { "radioPlugin" + pluginDescription.name }
).addChild("i", pluginDescription.getLocalisedPluginName());
```
or even
```java
option.addChild("label",
  new String[] { "for" },
  new String[] { "radioPlugin" + pluginDescription.name }
).addChild("i",
  pluginDescription.getLocalisedPluginName()
);
```
I saw a bit of everything around, so I had a hard time picking the favored choice, I rather tried to be more or less the same as what was nearby

Reply:
I’d say the second: that is what the code around it does.

Reply:
Done in f4982038679d841d84e02e8fdd19838f8e833d81